src/main/java/com/puppycrawl/tools/checkstyle/checks/naming/ConstantNameCheck.java

```diff
@@ -61,6 +61,26 @@
  * </pre>
  *
  * <p>
+ * Code Example:
+ * </p>
+ * <pre>
+ * class MyClass {
+ *   final static int logMYSELF = 10; // violation, name 'logMYSELF' must match
+ *                                    // pattern '[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'
+ *   final static int MYSELF = 100; // OK
+ *   final static int yourself = 1; // violation, name 'yourself' must match pattern
+ *                                  // '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'
+ *   public final static int FIRST = 100; // OK
+ *   public final static int second = 1; // violation, name 'second' must match pattern
+ *                                       // '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'
+ *   protected final static int THIRD = 100; // OK
+ *   protected final static int fOUR = 1; // violation, name 'fOUR' must
+ *                                        // match pattern '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'
+ *   private final static int FIVE = 100; // OK
```

Comment:
Please rename one of the constants using underscore and numbers as well. To show that default pattern allows such names